### ISSUE TITLE
Could not pass event PlayerPickupArrowEvent

### ISSUE THREAD

**Issue by mibby** [slipcor/pvparena#235]
```
[22:39:11] [Server thread/ERROR]: Could not pass event PlayerPickupArrowEvent to pvparena v1.3.3.185
org.bukkit.event.EventException
	at org.bukkit.plugin.EventExecutor$1.execute(EventExecutor.java:46) ~[patched_1.10.2.jar:git-Paper-823]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:74) ~[patched_1.10.2.jar:git-Paper-823]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[patched_1.10.2.jar:git-Paper-823]
	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:517) [patched_1.10.2.jar:git-Paper-823]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:502) [patched_1.10.2.jar:git-Paper-823]
	at net.minecraft.server.v1_10_R1.EntityArrow.d(EntityArrow.java:453) [patched_1.10.2.jar:git-Paper-823]
	at net.minecraft.server.v1_10_R1.EntityHuman.c(EntityHuman.java:412) [patched_1.10.2.jar:git-Paper-823]
	at net.minecraft.server.v1_10_R1.EntityHuman.n(EntityHuman.java:404) [patched_1.10.2.jar:git-Paper-823]
	at net.minecraft.server.v1_10_R1.EntityLiving.m(EntityLiving.java:1840) [patched_1.10.2.jar:git-Paper-823]
	at net.minecraft.server.v1_10_R1.EntityHuman.m(EntityHuman.java:156) [patched_1.10.2.jar:git-Paper-823]
	at net.minecraft.server.v1_10_R1.EntityPlayer.k_(EntityPlayer.java:257) [patched_1.10.2.jar:git-Paper-823]
	at net.minecraft.server.v1_10_R1.PlayerConnection.E_(PlayerConnection.java:139) [patched_1.10.2.jar:git-Paper-823]
	at net.minecraft.server.v1_10_R1.NetworkManager.a(NetworkManager.java:233) [patched_1.10.2.jar:git-Paper-823]
	at net.minecraft.server.v1_10_R1.ServerConnection.c(ServerConnection.java:152) [patched_1.10.2.jar:git-Paper-823]
	at net.minecraft.server.v1_10_R1.MinecraftServer.D(MinecraftServer.java:886) [patched_1.10.2.jar:git-Paper-823]
	at net.minecraft.server.v1_10_R1.DedicatedServer.D(DedicatedServer.java:404) [patched_1.10.2.jar:git-Paper-823]
	at net.minecraft.server.v1_10_R1.MinecraftServer.C(MinecraftServer.java:725) [patched_1.10.2.jar:git-Paper-823]
	at net.minecraft.server.v1_10_R1.MinecraftServer.run(MinecraftServer.java:624) [patched_1.10.2.jar:git-Paper-823]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_102]
Caused by: java.lang.NullPointerException
	at net.slipcor.pvparena.modules.powerups.PowerupManager.isPowerup(PowerupManager.java:325) ~[?:?]
	at net.slipcor.pvparena.modules.powerups.PowerupManager.onPlayerPickupItem(PowerupManager.java:372) ~[?:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor339.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor$1.execute(EventExecutor.java:44) ~[patched_1.10.2.jar:git-Paper-823]
	... 18 more
```
```
[22:39:11] [Server thread/ERROR]: Could not pass event PlayerPickupArrowEvent to pvparena v1.3.3.185
org.bukkit.event.EventException
	at org.bukkit.plugin.EventExecutor$1.execute(EventExecutor.java:46) ~[patched_1.10.2.jar:git-Paper-823]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:74) ~[patched_1.10.2.jar:git-Paper-823]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[patched_1.10.2.jar:git-Paper-823]
	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:517) [patched_1.10.2.jar:git-Paper-823]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:502) [patched_1.10.2.jar:git-Paper-823]
	at net.minecraft.server.v1_10_R1.EntityArrow.d(EntityArrow.java:453) [patched_1.10.2.jar:git-Paper-823]
	at net.minecraft.server.v1_10_R1.EntityHuman.c(EntityHuman.java:412) [patched_1.10.2.jar:git-Paper-823]
	at net.minecraft.server.v1_10_R1.EntityHuman.n(EntityHuman.java:404) [patched_1.10.2.jar:git-Paper-823]
	at net.minecraft.server.v1_10_R1.EntityLiving.m(EntityLiving.java:1840) [patched_1.10.2.jar:git-Paper-823]
	at net.minecraft.server.v1_10_R1.EntityHuman.m(EntityHuman.java:156) [patched_1.10.2.jar:git-Paper-823]
	at net.minecraft.server.v1_10_R1.EntityPlayer.k_(EntityPlayer.java:257) [patched_1.10.2.jar:git-Paper-823]
	at net.minecraft.server.v1_10_R1.PlayerConnection.E_(PlayerConnection.java:139) [patched_1.10.2.jar:git-Paper-823]
	at net.minecraft.server.v1_10_R1.NetworkManager.a(NetworkManager.java:233) [patched_1.10.2.jar:git-Paper-823]
	at net.minecraft.server.v1_10_R1.ServerConnection.c(ServerConnection.java:152) [patched_1.10.2.jar:git-Paper-823]
	at net.minecraft.server.v1_10_R1.MinecraftServer.D(MinecraftServer.java:886) [patched_1.10.2.jar:git-Paper-823]
	at net.minecraft.server.v1_10_R1.DedicatedServer.D(DedicatedServer.java:404) [patched_1.10.2.jar:git-Paper-823]
	at net.minecraft.server.v1_10_R1.MinecraftServer.C(MinecraftServer.java:725) [patched_1.10.2.jar:git-Paper-823]
	at net.minecraft.server.v1_10_R1.MinecraftServer.run(MinecraftServer.java:624) [patched_1.10.2.jar:git-Paper-823]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_102]
Caused by: java.lang.NullPointerException
	at net.slipcor.pvparena.modules.powerups.PowerupManager.isPowerup(PowerupManager.java:325) ~[?:?]
	at net.slipcor.pvparena.modules.powerups.PowerupManager.onPlayerPickupItem(PowerupManager.java:372) ~[?:?]
	at net.slipcor.pvparena.loadables.ArenaModuleManager.onPlayerPickupItem(ArenaModuleManager.java:209) ~[?:?]
	at net.slipcor.pvparena.listeners.PlayerListener.onPlayerPickupItem(PlayerListener.java:748) ~[?:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor262.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor$1.execute(EventExecutor.java:44) ~[patched_1.10.2.jar:git-Paper-823]
	... 18 more
```

**Comment by slipcor** [comment]
https://github.com/slipcor/pvparena_modules/blob/master/m_PowerUps/src/net/slipcor/pvparena/modules/powerups/PowerupManager.java#L335

Are you on the latest module version ? This can not possibly cause a NPE oO

**Comment by mibby** [comment]
I have ```Powerups - v1.3.2.59```. Same version as that java file since it was last touched. ```	v1.3.2.59 - fix Powerup config hook, add warning if misconfigured```

**Comment by mibby** [comment]
This error is so spammy in my logs. ;-;

**Comment by slipcor** [comment]
v1.3.3.185 - [PowerUps] - fix NPE, address github issue #235

I checked line 335 instead of 325 :P

**Comment by mibby** [comment]
Confirmed fixed. Can't seem to reproduce.
